ui/src/alerts/components/AlertsTable.tsx

```diff
@@ -172,7 +190,7 @@ class AlertsTable extends PureComponent<Props, State> {
             className={this.sortableClasses('time')}
             style={{width: colTime}}
           >
-            Time <span className="icon caret-up" />
+            Time ({timeZone})<span className="icon caret-up" />
```

Comment:
may be a nit but TimeZones.UTC is lowercase 'utc', and UTC is usually capitalized, just jumped out at me

Reply:
im okay with the constants all having the same format (lowercase) and the components / css deciding on how to display them